simulator/src/resim/cmd_publish.rs

```diff
@@ -47,27 +48,50 @@ impl Publish {
             let manifest = decompile(&transaction).map_err(Error::DecompileError)?;
             return fs::write(path, manifest).map_err(Error::IOError);
         }
+        self.store_package(out, &code)
+    }
+
+    pub fn publish_wasm<O: std::io::Write>(&self, out: &mut O) -> Result<(), Error> {
+        // Load wasm code
+        println!("Publishing ..");
+        let wasm_path = "wasm_dir/cur.wasm" ;
+        let code = fs::read(wasm_path).map_err(Error::IOError)?;
+        println!("Read code to variable");
+        self.store_package(out, &code)
+    }
```

Comment:
Do we need `publish_wasm`? Can we move it to PTE server? If not, can we make the path a parameter?

Reply:
Fixed that. Thanks!